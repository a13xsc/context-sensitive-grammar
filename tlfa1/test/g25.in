 (               {     =        ,      L       ,   F          ,   z,          ;  , N ,         W   ,         q      ,         n         ,   j   ,          X ,  s,  A      ,      &     , T          ,        G          ,  .        ,     /         ,   m ,     `      ,          +    ,  Q    ,     K ,       0  ,  H ,     I   ,     O         ,         R ,  d , ^  ,J , V   ,        E        ,      M  ,          C       ,         U  ,         c      ,  S         , 4      ,    P         ,y   ,          Y          ,        Z         ,   D               }       ,            {          +,       4,      0        , s          ,     n     ,  ^        ,    z          ,       q         ,  .  ,     y    ,    =       ,         `    ,  ;    ,    c    ,    m,          /        , d         ,    j      ,       &        }        ,               {             (   A    ,    zFLHOM+;.ZVYZ^MX.         )       ,      (      A    ,        =A=Tdj/RA`cSY&dFd`O          )   ,       (       A        ,         c^j4mj+nm04.cym0;s`          )   ,         (        C,         TTPITIEETEPIT         )    ,  (     C   ,;GTPU`       )        ,  (     E    ,         PPPTIPTI      )      ,         ( E         ,       TyszU        )       ,      (   E   ,       IRcVD^XAJ0dEdMT     )          ,    (     I  ,          PPPPPPPPPP   ) ,         (         H  ,        RD0G),          (       K       ,QzdAS  )         ,     (  K   ,+;d;=+q4^          )   ,    (          J         ,      K&   )       ,   (       J        ,ZCd^P;RPEVzIA&j       ) ,    (  J ,       `^ds^qq+yq0dj+.+&     )   ,   (       M ,     CFS0   )    ,         (   P          ,     WGTTc          )  ,        (          P  ,      dQGcC.DHTcmD`G;+0.d     )        ,   (  P       ,      e)   , (S          , PU^cWjRHUP       )    ,         (          T,   IIIPIIPII       )        , (       Y     ,       KYd+^InRjm`JmD;Q.=sQ         )     ,      (Y  ,          XNF=dY`      )     ,  (   X    ,          0/ssnq0dq          )        ,          (   Z      ,   ^;//&`=/q4/     )        }           ,          D         )      
