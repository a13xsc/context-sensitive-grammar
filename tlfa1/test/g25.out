V
L
H
U
M
Y
D
O
Q
F
G
W
R
S
N
